Dockerfile

```diff
@@ -1,13 +1,16 @@
-FROM python:3.9-slim
+FROM node:20.13.1-bookworm-slim
 
 WORKDIR /telemetry-api
 
-COPY requirements.txt requirements.txt
 
-RUN pip install --no-cache-dir -r requirements.txt
+COPY package*.json ./
+
+RUN npm install
 
 COPY . .
 
 EXPOSE 5000
 
-CMD ["python","telemetry.py"]
\ No newline at end of file
+ENV PORT=5000
+
+CMD ["node","telemtry.js"]
```

Comment:
I think if you want hot-reload for development, you'll need something like nodemon. But only for dev containers (not prod)

Reply:
Also, I think the filename is supposed to be `telemetry.js`